sdk/keyvault/azure-security-keyvault-keys/src/delete_key_operation.cpp

```diff
@@ -54,16 +54,11 @@ Azure::Security::KeyVault::Keys::DeleteKeyOperation::DeleteKeyOperation(
     Azure::Response<Azure::Security::KeyVault::Keys::DeletedKey> response)
     : m_pipeline(keyvaultPipeline)
 {
-  if (!response.HasValue())
-  {
-    throw Azure::Security::KeyVault::Common::KeyVaultException(
-        "The response does not contain a value.");
-  }
```

Comment:
@vhvb1989, if such a scenario does come up, where you need this check, please let me know and add a test. Looking at existing tests and code coverage, this code-block and exception was never hit.